tickets/src/utils/dates.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * This function returns the future days, months and up to 3 years based on a current
+ * date.
+ * @param {*} now
+ * @param {*} year
+ * @param {*} month
+ */
+export const getDaysMonthsAndYears = (now, year, month) => {
+  let monthNames = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+  ]
+
+  // Years are the next 3 years
+  const years = [
```

Comment:
If you wanted to compress this a bit, the following should be equivalent. Just an option
```javascript
const years = [0, 1, 2].map(n => (now.getFullYear() + n).toString
// => [ '2019', '2020', '2021' ]
```